js/views/itemEditVw.js

```diff
@@ -111,7 +111,7 @@ module.exports = Backbone.View.extend({
     if(priceType === "local"){
       inputParent.find('.js-priceLocal').val(inputParent.find('.js-priceBtc').val() * window.currentBitcoin);
     }else{
-      inputParent.find('.js-priceBtc').val(inputParent.find('.js-priceLocal').val() / window.currentBitcoin);
+      inputParent.find('.js-priceBtc').val(inputParent.find('.js-priceLocal').val() / window.currentBitcoin);  //TOOD: Eliminate division because of floating point math
```

Comment:
The floating point math isn't an issue here, since the result is a rounded up approximation.